notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
     "\n",
```

Comment:
<p>Also can this be 1 paragraph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/289/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Should I combine it with the other cells to make one larger paragraph? Otherwise, I think this cell and the one above are already singular paragraphs. </p>

Reply:
Maybe its just rendering weird for me on nbreviwer:
![image](https://user-images.githubusercontent.com/4494906/123457189-a9d26e00-d5b1-11eb-8dae-8d198831471c.png)

See at the bottom how it breaks the word "look" into a new paragraph?

Reply:
yeah, ignore this, it looks fine in the actual *ipynb